common/src/main/java/com/microsoft/hydralab/common/entity/agent/MobileDevice.java

```diff
@@ -33,8 +34,12 @@ public MobileDevice() {
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
```

Comment:
Could we consider allowing this style and disabling this rule entirely? I this the oneliner is quite useful and readable.

Reply:
We can remove `NeedBraces` in our checkstyle-ruleset.xml, but I personally don't like non-braces style xD